torchci/test/drciUtils.test.ts

```diff
@@ -334,6 +334,7 @@ describe("Test various utils used by Dr.CI", () => {
   });
 
   test("test isInfraFlakyJob", () => {
+    // Not a workflow job
```

Comment:
maybe im reading this wrong, but do you have any tests where it would have been flaky if it were a workflow job, but not if it is a workflow run?

Reply:
There are examples from the original PR https://github.com/pytorch/test-infra/pull/4622 that implements `isInfraFlakyJob`.  They were all jobs that weren't run.  GitHub records are gone, but they are still shown on HUD

Reply:
By querying Rockset, I can confirm that there were infra flaky jobs from 110608.  Here is an example:

```
workflowID = 6420960240
id = 17434762139
runnerName = ''
authorEmail = 'eltociear@gmail.com'
name = 'pull / linux-jammy-py3.8-gcc11 / test (distributed, 2, 2, linux.2xlarge)'
jobName = 'linux-jammy-py3.8-gcc11 / test (distributed, 2, 2, linux.2xlarge)'
conclusion = 'failure'
failure_captures = null
```

Reply:
Sorry what I mean is that I don't see tests where the existence of a workflowID has an effect on the output of the function, I only see cases that returned false regardless of whether or not the workflowID is there.  This makes me think that the original code would have also returned false, so I'm confused as to if the code works, am I missing something?